react/src/components/StorageStatusPanel.tsx

```diff
@@ -228,6 +233,19 @@ const StorageStatusPanel: React.FC<{
                 {t('data.ProjectFolder')}:
               </Typography.Text>
               {projectFolderCount}
+              {isAdmin &&
+              (project_resource_policy?.max_vfolder_count ||
+                project_resource_policy?.max_vfolder_count === 0) ? (
+                <>
```

Comment:
This condition handles `number` AND `>=0`
```suggestion
              (project_resource_policy?.max_vfolder_count ?? -1) >= 0 ? (
                <>
```